### ISSUE TITLE
Add methods to easily append .js and .css on page build

### ISSUE THREAD

**Issue by fredck** [benderjs/benderjs#216]
Currently we're passing strings with chunks of HTML to `data.parts` on page building to inject .js and .css files into `<head>`, which is cumbersome.

There should be a short easy way to do it through a simple API call instead.

**Comment by fredck** [comment]
Proposal:

```
data.js = []; // An array of js files URLs
data.css = []; // And array of css files URLs
```

There would be injected at the end of the `<head>` element.

**Comment by fredck** [comment]
@gregpabian pointed out that `data.js` is already in use so we have a conflict. The new proposal:

```
data.addJS( url, options );
data.addCSS( url, options );

// options:
{
    // Add the file at the end of <body> (defaults to <head>)
    body: true
}
```

**Comment by gregpabian** [comment]
Resolved with #222
